client/components/ZoopList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+
+const ZoopsList = () => {
+  return (
+    <>
+      <p>I am the Zoops List</p>
+      
+      <Outlet />
```

Comment:
Why is there an Outlet here? I don't think it's needed, this page shouldn't need a sub-route